frontend/pages/hosts/details/cards/Software/InstallStatusCell/InstallStatusCell.tsx

```diff
@@ -44,11 +45,27 @@ export const INSTALL_STATUS_DISPLAY_OPTIONS: Record<
       </>
     ),
   },
+  verifying: {
+    iconName: "success-outline",
+    displayText: "Verifying",
+    tooltip: () => (
+      // TODO
```

Comment:
TODO?

Reply:
Oh yeah, this is the part that's yours but I had to update some since the typings would break it